### ISSUE TITLE
Propel not working

### ISSUE THREAD

**Issue by AyindeAA** [propelorm/Propel2#1820]
I followed the [documentation](http://propelorm.org/documentation/01-installation.html) of the Propel2:
in the command line, I type propel init, it shows command does not exist in this namespace.
Am using php8.0
I have installed all dependencies using composer.
But still not working.
I love this package.

**Comment by mringler** [comment]
You have to pass "init" or "sql:build" as option to the propel command, i.e. enter`./propel init` instead of just `init`

**Comment by AyindeAA** [comment]
![Screenshot from 2022-01-08 07-49-25](https://user-images.githubusercontent.com/12680894/148634831-95900f8f-5e95-40a2-8bed-884296eeec07.png)
That is what I type it still shows the same thing.

**Comment by mringler** [comment]
Right, sorry, I misinterpreted your prompt. Does `propel list` work? What version do you get from `propel --version`?

**Comment by AyindeAA** [comment]
The version is Propel 2.0.0-dev.
My Php Version PHP8.0
![Screenshot from 2022-01-08 11-31-51](https://user-images.githubusercontent.com/12680894/148640965-2f8c3352-2025-41cd-bb53-43c7babf06bb.png)
propel list gives the output above.

**Comment by dereuromark** [comment]
Can we close this?
Doesnt look like an issue with propel itself.
